src/pybind/mgr/dashboard/module.py

```diff
@@ -7,16 +7,39 @@
 import errno
 import os
 import socket
+import tempfile
+import threading
+from uuid import uuid4
+from OpenSSL import crypto
+
 try:
     from urlparse import urljoin
 except ImportError:
     from urllib.parse import urljoin
+
 try:
     import cherrypy
 except ImportError:
     # To be picked up and reported by .can_run()
     cherrypy = None
 
+
+if cherrypy is not None and cherrypy.__version__ == "3.5.0":
```

Comment:
Do you know what version it was fixed in? I wonder if checking for 3.5.0 only is too narrow? How about "if version is lower than <version we know this was fixed in>"?

Reply:
Turns out it was fixed in 3.7.0, so I've updated this to be a range (think it's very unlikely anyone's going to put 3.6.0 in the wild but people are crazy so who knows!)